src/components/CardModal/CardModalAttachments/CardModalAttachment.js

```diff
@@ -0,0 +1,164 @@
+                <p>Attachments</p>
+            </div>
+            { files.map( attachment =>
+                <div key={attachment.item.fileId} className=" flex mb-3  hover:bg-black hover:bg-opacity-5 rounded">
+                    <div className='flex items-center'>
+                        <div className='flex w-32 h-24 items-center justify-center rounded overflow-hidden' style = {{backgroundColor: attachment.color }}> 
+                            <a className="max-w-32 max-h-24 rounded" download={attachment.item.name} href = {attachment.file} >
+                                { attachment.isImage ? (
+                                    <img className = "max-w-32 max-h-24" src = {attachment.file} />
+                                ) : (
+                                    <span className="flex text-lg font-bold text-gray-600">{attachment.fileExtension}</span>
+                                )
+                                }
+                            </a>
+                        </div>
+                    </div>
+                    <div className="flex w-full flex-col text-gray-600 pl-3 rounded-ibs text-sm">  
+                        <span className="font-bold break-all pt-1.5">
+                            {attachment.item.name}
+                        </span>
+                        <span className="break-words">
+                            <span dt={ attachment.item.date}>{ getRelativeDate(attachment.item.date)} </span>
+                            <span> - <span className="underline cursor-pointer">Comment</span></span>
+                            <span> - <span className="underline cursor-pointer" onClick= {() => handleShowAttachmentDeleteModal(attachment.item.fileId)}>Delete</span></span>
+                            <span> - <span className="underline cursor-pointer" onClick= {() => handleShowAttachmentEditModal(attachment.item.fileId)}>Edit</span> </span>
+                        </span>
+                    </div>
+                    { (attachmentDeleteMenuOpen.status && attachmentDeleteMenuOpen.id===attachment.item.fileId)
+                        && <AttachmentDeleteMenu id={attachment.item.fileId}/>
+                    }
+
+                    { (attachmentEditMenuOpen.status && attachmentEditMenuOpen.id===attachment.item.fileId)
+                        && <AttachmentEditMenu id={attachment.item.fileId} name={attachment.item.name}/>
+                    }
+                </div>
+            )}
+
+            <button
+                className="gap-2 py-1 px-2 mb-2 w-44  text-center bg-trello-gray-card-modal-buttons hover:bg-trello-gray-card-modal-buttons-hover text-trello-blue-card-modal-button-text items-center text-base shadow-sm rounded-sm"
+                onClick={()=> handleShowAttachmentMenuModal(2)}
+            >
+                <p>Add an attachment</p>                
+            </button>
+            { (attachmentMenuOpen.status && attachmentMenuOpen.id===2) 
```

Comment:
some of your formatting is a bit off. have you got prettier set up?

Reply:
Nope I have not. I've set it up and have formatted the page